media/css/mozorg/rise25/components/category.scss

```diff
@@ -0,0 +1,145 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@use '../lib' as r25;
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+// * -------------------------------------------------------------------------- */
+// Nomination categories
+
+.r25-c-category {
+    @include text-body-lg;
+    border-top: 1px solid fade-out($color-white, 0.8);
+    cursor: pointer;
+    padding: $layout-md 0;
+
+    &:last-child {
+        border-bottom: 1px solid fade-out($color-white, 0.8);
+    }
+
+    .r25-c-category-body {
+        max-width: $content-lg;
+    }
+
+    .r25-c-category-desc {
+        @include text-body-xl;
+        margin: 0;
+    }
+
+    .r25-c-category-expand {
+        margin: $spacing-lg 0 0;
+    }
+
+    .r25-c-category-cta {
```

Comment:
**issue:** On mobile screen sizes, the Nominate buttons could use some space between the copy above. The buttons also could maybe stand out more here, since on touch screens you don't get the hover highlight effect?

![image](https://github.com/mozilla/bedrock/assets/400117/872c2d7a-b889-4c8f-9106-e06af314c833)

Reply:
I agree, but this is as designed (I actually added the full border; the design only had a subtle highlight on the top). I'll see what Ryan thinks and maybe we can make them look more buttony.